src/styles/sourcesHeader.scss

```diff
@@ -0,0 +1,3 @@
+.pf-v5-c-content {
```

Comment:
What is this rule required for? We should probably make it more specific to target correct element.

Reply:
This is so that the subtitle of the new header more closely matches what is shown in the mocks, and does not spill underneath the button. I've updated the `SourcesHeader` component to pass in an `ouiaId`, and we can use that selector to specifically target the subtitle text in this css.

https://github.com/RedHatInsights/sources-ui/pull/1219/commits/a57262375b29a772cfa3b283615f7094abb95f70